### PR TITLE
#5 Show latest event and about soiree combined

### PR DESCRIPTION
**Pull request by shubhamnautiyal**

### CODE REVIEW

File: public/wp-content/themes/ColoredCow/functions.php [hunk 1]
```diff
@@ -14,20 +16,15 @@ function cc_scripts() {
...
+        wp_enqueue_style('cc-custom-font','https://fonts.googleapis.com/css?family=Luckiest+Guy|Marcellus+SC|Margarine|Oswald|Paprika|Roboto');
         wp_enqueue_style('style', get_template_directory_uri().'/style.css');
-
         
-
     }
     add_action('wp_enqueue_scripts','cc_styles');
 }
-
 //add filter to remove margin above html
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/functions.php]: remove coment

File: public/wp-content/themes/ColoredCow/landing.php [hunk 2]
```diff
@@ -4,10 +4,75 @@
...
 ?>
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-12 col-md-12 col-lg-6">
+				<div class="soiree-title">Soiree</div>
+			  	<p class="soiree-content">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+				<hr>
+				<div class="request-invite">
+						<div>Want to join the party?</div>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: indentation

File: public/wp-content/themes/ColoredCow/src/scss/fonts.scss [hunk 3]
```diff
@@ -1,3 +1,7 @@
 .oswald{
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/fonts.scss]: name it as .font-oswald{...}

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 4]
```diff
@@ -37,16 +37,66 @@ Text Domain: coloredcow
...
+.modal-title{ 
+	@extend .marcellus;
+	font-size: 25px;
+}
+
+.modal-body{
+	background-color: $whitesmoke;
+}
+
+#footer{
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: use class instead of id

**Review by Shubham2301 — CHANGES_REQUESTED** [review]

File: public/wp-content/themes/ColoredCow/src/scss/fonts.scss (hunk 3 above)
Comment (shubhamnautiyal) [reply public/wp-content/themes/ColoredCow/src/scss/fonts.scss]: done

File: public/wp-content/themes/ColoredCow/landing.php (hunk 2 above)
Comment (shubhamnautiyal) [reply public/wp-content/themes/ColoredCow/landing.php]: corrected

File: public/wp-content/themes/ColoredCow/functions.php (hunk 1 above)
Comment (shubhamnautiyal) [reply public/wp-content/themes/ColoredCow/functions.php]: done

**Pull request merged**